.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: build
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        go-version: [ 1.13.x ]
+        os: [ ubuntu-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
+          args: --issues-exit-code=0
+      - name: test
+        run: make build
```

Comment:
I think this is one improvement we could make here, can we split build into 

`make test`

`make tester`

so that when it fails, it is more clear in the CI logs about which step failed.

Otherwise 👍

Reply:
done, thank you 🙏